packages/react-next/src/index.ts

```diff
@@ -20,6 +20,7 @@ export * from './DetailsList';
 export * from './Dialog';
 export * from './Divider';
 export * from './DocumentCard';
+export * from './DragDrop';
```

Comment:
missing the same change in `@fluentui/react` package

Reply:
The `@fluentui/react` package does this:

```
import './version';
export * from 'office-ui-fabric-react';
```